superset-frontend/src/visualizations/FilterBox/FilterBox.jsx

```diff
@@ -409,37 +439,56 @@ class FilterBox extends React.PureComponent {
     return filtersFields.map(filterConfig => {
       const { label, key } = filterConfig;
       return (
-        <div key={key} className="m-b-5 filter-container">
+        <StyledFilterContainer key={key} className="filter-container">
```

Comment:
This className is used by other less files. We should probably remove it once all the less files have been removed from the codebase